StyleCop.Analyzers/StyleCop.Analyzers.Test/MaintainabilityRules/SA1404UnitTests.cs

```diff
@@ -110,6 +166,18 @@ public void Bar()
             DiagnosticResult expected = this.CSharpDiagnostic().WithLocation(3, 66);
 
             await this.VerifyCSharpDiagnosticAsync(testCode, expected, CancellationToken.None).ConfigureAwait(false);
+
+            var fixTestCode = @"public class Foo
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage(null, null, Justification = """ + SA1404CodeAnalysisSuppressionMustHaveJustification.JustificationPlaceholder + @""")]
+    public void Bar()
+    {
+
+    }
+}";
+
+            // Do not verify diagnostics as the placeholder will still generate a violation
```

Comment:
:bulb: You could still verify the `fixTestCode` diagnostics, it just won't be empty like we normally have.